backend/src/main/java/pl/edu/pw/ee/backend/api/order/BazaZamowien.java

```diff
@@ -43,60 +39,32 @@ public int setOrderData(OrderDTO orderData) {
 
         log.info("Retrieving client with id: {}", orderDataDto.clientId());
 
-        Client client = clientRepository.findById(orderDataDto.clientId())
-                .orElseThrow(() -> new ClientNotFoundException(orderDataDto.clientId()));
+        Client client = clientService.findClientById(orderDataDto.clientId());
 
         OrderData orderDataToSave = buildOrderDataToSave(orderDataDto, client);
```

Comment:
Wydaje mi się że to powinno być wykonane w serwisie przy zapisaniu ordera ale szczerze nwm. @seredak319 jak sądzisz?